dask_expr/expr.py

```diff
@@ -731,6 +731,7 @@ class Blockwise(Expr):
 
     operation = None
     _keyword_only = []
+    projection_passthrough = False
```

Comment:
Final comment (I promise): Should we make this private?

Reply:
Fine by me. Makes probably more sense here